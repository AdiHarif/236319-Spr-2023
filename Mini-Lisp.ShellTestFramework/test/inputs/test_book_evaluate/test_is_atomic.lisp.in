(is-atomic 'car)
(is-atomic 'ooo)
