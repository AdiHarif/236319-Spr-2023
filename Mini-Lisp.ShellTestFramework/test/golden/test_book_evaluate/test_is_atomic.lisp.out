T
T
T
T
T
T
T
T
T
NIL
NIL
